apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/config/ChainsConfigLoader.java

```diff
@@ -20,37 +25,43 @@
 
     public ChainsConfigLoader(boolean ignoreResources, String configDir, String resourceName) {
         super(ignoreResources, configDir, resourceName);
+        MAPPER.registerModule(new ParanamerModule());
```

Comment:
Single responsibility violated.  ChainsConfigLoader dont care about jackson and Paranamer module

Reply:
The call to **registerMapper()** is moved inside unit tests only.